src/main/java/me/ryanhamshire/GriefPrevention/GriefPrevention.java

```diff
@@ -3614,7 +3594,12 @@ public boolean pvpRulesApply(World world)
 
     public static boolean isNewToServer(Player player)
     {
-        if(player.getStatistic(Statistic.PICKUP, Material.WOOD) > 0) return false;
+        if(     player.getStatistic(Statistic.PICKUP, Material.OAK_PLANKS) +
+                player.getStatistic(Statistic.PICKUP, Material.SPRUCE_PLANKS) +
+                player.getStatistic(Statistic.PICKUP, Material.BIRCH_PLANKS) +
+                player.getStatistic(Statistic.PICKUP, Material.JUNGLE_PLANKS) +
+                player.getStatistic(Statistic.PICKUP, Material.ACACIA_PLANKS) +
+                player.getStatistic(Statistic.PICKUP, Material.DARK_OAK_PLANKS) > 0) return false;
```

Comment:
Probably should include this in the PR description next time. Thanks though, been planning to "fix" this for some time now.

Reply:
Shouldn't it be LOG  variants, not PLANKS?